<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SheetTest</class>
 <widget class="QWidget" name="SheetTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>488</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SheetTest</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="lb1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>选中模式</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbModel">
        <property name="minimumSize">
         <size>
          <width>380</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>最小选中粒度</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbGrainSize">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectOnOff">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>转换选中</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="allOnOff">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>全点亮</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tw">
     <property name="maximumSize">
      <size>
       <width>1200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
